src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +614,115 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
 ```bash
 Successfully updated the mesh with the id: MESH_ID
 ```
+
+## aio api-mesh:log-list
+
+The `log-list` command lists the last 15 events for your mesh by rayID.
```

Comment:
Should update section. `log-list` command lists the most recent requests for your mesh by rayID. By default this command shows the 15 most recent requests. 

Then in the --filename section, add sentence. This flags exports all rayIDs from the past 15 minutes.